### ISSUE TITLE
Unity 5 Demo broadcasting to Resolume Arena 4 – Can't view Unity Demo output

### ISSUE THREAD

**Issue by dwanyeeast** [Syphon/Unity3D#22]
I was able to import the Unity Syphon package successfully in Unity 5 demo. I added the Syphon.cs and the SyphonServerTexture to my Player 1 and Player 2 cameras and ran my Unity game.

In Resolume Arena 4, with Syphon Input on, I can see Unity is broadcasting, but when I drag each of the broadcasts to my composition I see **absolutely nothing**.

Is this result because I don't have a Unity Pro version? Is it imperative I do or is there something I'm not doing correctly?

Can anyone shed some light on this, please?

**Comment by vade** [comment]
Use OpenGL 2.1 renderer. please see the closed / existing git issues.

**Comment by dwanyeeast** [comment]
@vade 
Thanks for the input. I did review the closed/existing git issues, and I even changed the project settings to use OpenGL 2.1 (OpenGL2), but I'm still not seeing the Unity game displayed in Resolume Arena. I see it being broadcast in Resolume's Source tab, but I get a blank display.

Is this result because I'm not running a pro version of Unity 5.3?

**Comment by vade** [comment]
Yes. Unity Pro is required for plugin support as far as I know. This has been consistent with every version of Unity.

**Comment by brianchasalow** [comment]
@vade this actually isn't true. Unity 'Personal' (as of Unity 5) now supports RenderTextures / Image Effects / supports everything Pro supports. Not sure why you'd be seeing black, though. Maybe alpha issues? Try putting the Syphon Alpha One script on the camera?

**Comment by vade** [comment]
Non-Pro It supports plugins? What makes it Pro then? Sorry for the mis-info! Should change the readme / main page of the repo then.

**Comment by dwanyeeast** [comment]
@brianchasalow is correct. I also identified why I was seeing black. In Unity, the Resolume Arena name in the Syphon Script has to be highlighted.

I now see my game broadcasting in Resolume. Thanks, mate! This saved me $79USD/month

I now hit a second problem:

I can only see from one camera and not the other. I setup 2 cameras in Unity, but only one is broadcasting in Resolume.

I **Syphon Server Texture** on both Player 1 Camera and Player 2 Camera, but the **Syphon Script** component is on Player 1 Camera.

In Resolume, only Player 2 Camera is broadcasting.

In Unity, in the Syphon Script component (while in Game-mode), I see an additional drop down with Player 1 Camera and Player 2 Camera. Player 1 Camera is selected. It won't let me choose Player 2 Camera.

Any thoughts?
